web/src/components/planner/PlannerFactorySatisfaction.vue

```diff
@@ -120,6 +121,16 @@
                 </span>
               </v-chip>
             </div>
+          </v-card-text>
+        </v-card>
+        <v-card class="sub-card border-md">
```

Comment:
They can be in the same `<v-card-text>` you can just add one of these between the divs:

`<v-divider color="#ccc" thickness="2px" />`

This will add a nice line seperating the two sections.